File: Ejercicio_0026/Capitalize.java
package Ejercicio_0026;

import javax.swing.JOptionPane;

public class Capitalize {
    public static void main(String[] args) {
        String s = JOptionPane.showInputDialog("\nEscribe la cadena:\n");

        s += " ";
        String mesage = "";
        String texto = "";

        for (int i = 0; i < s.length(); i++) {
            if (s.charAt(i) == ' ') {
                mesage += texto.substring(0, 1).toUpperCase() + texto.substring(1).toLowerCase() + " ";
                texto = "";
            } else {
                texto += s.charAt(i);
            }
        }

        JOptionPane.showMessageDialog(null, "\nResultado:  " + mesage + "\n");
    }
}

File: Ejercicio_0026/Capitalize.js
function solvee(s) {
    s += " ";
    let mesage = "";
    let texto = "";

    for (let i = 0; i < s.length; i++) {
        if (s[i] === " "){
            mesage += texto.charAt(0).toUpperCase() + texto.slice(1).toLowerCase() + " ";
            texto = "";
        } else {
            texto += s[i];
        }
    }

    alert("\nResultado:  " + mesage + "\n")
}

let s = prompt("\nEscribe la cadena:\n")

solvee(s)